mongodbatlas/performance_advisor.go

```diff
@@ -0,0 +1,230 @@
+package mongodbatlas
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+)
+
+const (
+	performanceAdvisorPath                     = "groups/%s/processes/%s/performanceAdvisor"
+	performanceAdvisorNamespacesPath           = performanceAdvisorPath + "/namespaces"
+	performanceAdvisorSlowQueryLogsPath        = performanceAdvisorPath + "/slowQueryLogs"
+	performanceAdvisorSuggestedIndexesLogsPath = performanceAdvisorPath + "/suggestedIndexes"
+)
+
+// PerformanceAdvisorService is an interface of the Performance Advisor
+// endpoints of the MongoDB Atlas API.
+//
+// See more: https://docs.atlas.mongodb.com/reference/api/performance-advisor/
+type PerformanceAdvisorService interface {
+	GetNamespaces(context.Context, string, string, *NamespacesOptions) (*Namespaces, *Response, error)
+	GetSlowQueries(context.Context, string, string, *SlowQueriesOptions) (*SlowQueries, *Response, error)
+	GetSuggestedIndexes(context.Context, string, string, *SuggestedIndexesOptions) (*SuggestedIndexes, *Response, error)
```

Comment:
[grammar] only the last word as plural
```suggestion
	GetNamespaces(context.Context, string, string, *NamespaceOptions) (*Namespaces, *Response, error)
	GetSlowQueries(context.Context, string, string, *SlowQueryOptions) (*SlowQueries, *Response, error)
	GetSuggestedIndexes(context.Context, string, string, *SuggestedIndexOptions) (*SuggestedIndexes, *Response, error)
```